Core/Utilities/SiteSEMapping.py

```diff
@@ -8,10 +8,78 @@
 
 __RCSID__ = "$Id$"
 
-from DIRAC import S_OK
+import errno
+
+from DIRAC import gLogger, S_OK, S_ERROR
 from DIRAC.DataManagementSystem.Utilities.DMSHelpers import DMSHelpers, siteGridName
 
 
+def getSEParameters(seName):
+  """ get all the SE parameters in a list
+
+      :param str seName: name of the Storage Element
+
+      :return: S_OK() or S_ERROR
+  """
+  # This import is here to avoid circular imports
+  from DIRAC.Resources.Storage.StorageElement import StorageElement
+  se = StorageElement(seName, hideExceptions=True)
+
+  protocolsSet = set(se.localAccessProtocolList) | set(se.localWriteProtocolList)
+
+  seParametersList = []
+  for protocol in protocolsSet:
+    seParameters = se.getStorageParameters(protocol=protocol)
+    if seParameters['OK']:
+      seParametersList.append(seParameters['Value'])
+    else:
```

Comment:
I would just `continue` because you will often have in the protocol list some protocols that you do not provide, and of course that can't work. (typically, dips).

Reply:
Done.